.github/workflows/generated_files_sync.yaml

```diff
@@ -20,7 +20,7 @@ on:
   pull_request:
 name: generation diff
 env:
-  library_generation_image_tag: latest
+  library_generation_image_tag: 2.39.0
```

Comment:
Is this configured to be updated renovate bot?

Reply:
I plan to update renovate config in separate PR.